docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-stability-and-performance/database-tuning-for-upgrades.md

```diff
@@ -46,4 +46,6 @@ The default configuration works well. It configures [asynchronous I/O to disk](h
 
 Turn off [synchronous commits](https://www.postgresql.org/docs/10/wal-async-commit.html) and set the [write ahead log writer delay](https://www.postgresql.org/docs/10/wal-async-commit.html) to `1000` milliseconds.
 
-Configurations on tuning your database for upgrading. When you're done upgrading, make sure tune your database for production again.
\ No newline at end of file
+## Conclusion
+
+When you're done upgrading, make sure revert your database configurations back to their production settings.
```

Comment:
Typo fix: change "make sure revert" to "make sure to revert". I forgot to put "to" in the original text. :-D